.github/workflows/test-all.yml

```diff
@@ -39,6 +40,11 @@ jobs:
     - name: install Chrome stable
       run: |
         npx @puppeteer/browsers install chrome@stable
+        chromeVersionString=$(ls chrome)
+        if [ "$CHROME_VALIDATED_VERSION" != "$chromeVersionString" ]; then 
```

Comment:
Will this stand out in the workflow in any way if the tests fail? I'm guessing contributors will look at the failed step further down if tests fail, is there anything to direct them to look at this one?